CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features
 
+- Updated message explaining the node syncing process on the initial screen ([PR 2831](https://github.com/input-output-hk/daedalus/pull/2831))
```

Comment:
These two lines should be merged, please see the rest of changelog for examples

Reply:
Done :+1: